subprojects/docs/src/docs/release/notes.md

```diff
@@ -133,6 +133,22 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 
 Check the [version catalog API](javadoc/org/gradle/api/artifacts/VersionCatalog.html) for all supported methods.
 
+#### Control skipping Kotlin metadata version check for script compilation
+
+You can now control if Kotlin DSL script compilation should skip Kotlin metadata version check.
+Skipping Kotlin metadata version check in Kotlin DSL script compilation is now deprecated.
+
+To opt in to the future-proof behaviour Today, set the `org.gradle.kotlin.dsl.skipMetadataVersionCheck` property to `false`.
```

Comment:
```suggestion
To opt-in early to this future-proof behavior, set the `org.gradle.kotlin.dsl.skipMetadataVersionCheck` property to `false`.
This will enable the metadata check.
```